src/app/find-help/transport-search/transport-search.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Pageable, TransportOfferSearchCriteria, TransportResourceService, TransportOffer } from '../../../api';
+
+@Component({
+  selector: 'app-transport-search',
+  templateUrl: './transport-search.component.html',
+  styleUrls: ['./transport-search.component.scss'],
+})
+export class TransportSearchComponent {
+  results: TransportOffer[] = [];
+  total?: number = undefined;
+  loading = false;
+  constructor(private http: HttpClient, private transportResourceService: TransportResourceService) {}
+
+  // See https://github.com/mmiszy/polish-plurals/blob/5d2dc0cdea93e68e0a320a0201f0acb7a53206d0/index.mjs
+  pluralizePolish(one: string, few: string, many: string, count: number) {
```

Comment:
An angular pipe would be more efficient AFAIK, as in contrary to method call, it's executed only when input changes.

Reply:
Makes sense, I'll update the code.